### PR TITLE
adding Markus formatting code

### PR DESCRIPTION
**Pull request by szelenka**
Adding placeholder for the code Markus worked on to get the formatting correct.

Still need to validate the "sides" will fit on the single row

### CODE REVIEW

File: src/triggers.js [hunk 1]
```diff
@@ -133,7 +133,9 @@ function notifySidebars() {
 
 function printLabel(order_id) {
   var worksheet = new Worksheet();
-  worksheet.validateAndAdvanceState(order_id,'Present');
+  if (worksheet.validateAndAdvanceState(order_id,'Present') === -1) {
```
Comment (bobcallaway) [inline src/triggers.js]: My thought is that we should print the label first; only if that is successful should we change the state to 'Labeled'... i push some quick code to master last night that cleaned this up a bit. thats probably the merge conflict for this branch.

File: src/printLabels.js [hunk 2]
```diff
@@ -111,19 +111,103 @@ function createLabelFileFromSheet(orderDetails) {
...
     printSuccessful = true;
   }
 
   return printSuccessful;
+}
+
+
+//https://ctrlq.org/code/20061-google-cloud-print-with-apps-script
+function getCloudPrintService() {
+  //TODO: replace CLIENT_ID/SECRET
```
Comment (bobcallaway) [inline src/printLabels.js]: We should probably put the resulting Bearer token into the [DocumentCache](https://developers.google.com/apps-script/reference/cache/cache-service#getDocumentCache()) for reuse. My guess is that it won't expire throughout the night and there's no reason to do the OAuth dance for every transaction.

File: src/printLabels.js [hunk 3]
```diff
@@ -85,19 +111,103 @@ function createLabelFileFromSheet(orderDetails) {
...
+      duplex: {
+        type: "NO_DUPLEX"
+      }
+    }
+  };
+
+  var payload = {
+    "printerid" : printerID,
+    "title"     : docName,
+    "content"   : DriveApp.getFileById(docID).getBlob(),
```
Comment (bobcallaway) [inline src/printLabels.js:189]: I saw [this](https://stackoverflow.com/questions/30565554/oauth2-with-google-cloud-print) stackoverflow post that indicated you could set mime-type to `google.kix` and pass the documentID vs converting it to a blob and sending it all over. Perhaps an optimization at this point but worth noting.

File: src/printLabels.js [hunk 4]
```diff
@@ -85,19 +111,103 @@ function createLabelFileFromSheet(orderDetails) {
...
+    "contentType": "google.kix",
```
Comment (bobcallaway) [inline src/printLabels.js:190]: if you do go with `getBlob()` above this may need to change to `application/pdf`

**Comment by szelenka** [comment]
Implementing changes in next PR

**Pull request merged**